list-sensors.cpp

```diff
@@ -167,6 +175,18 @@ class Properties : public PropertiesMap
         {
             ret = "N/A";
         }
+        else if (std::holds_alternative<double>(it->second))
+        {
+            auto value = std::get<double>(it->second);
+            if (value < 1000)
```

Comment:
Why 1000? Is it upstream's logic?

Reply:
No, it's our internal logic. All values with more than 3 digits should be shown as an integer value.
Probably here should be used `<=`.

Reply:
> Probably here should be used `<=`.

No, the current implementation is correct. All values less than 1000 will be shown as float with 3 digits after coma.
All other values will be shown as integer values.